docs/rules/camelcase.md

```diff
@@ -6,6 +6,23 @@ When it comes to naming variables, styleguides generally fall into one of two ca
 
 This rule looks for any underscores (`_`) located within the source code. It ignores leading and trailing underscores and only checks those in the middle of a variable name. If ESLint decides that the variable is a constant (all uppercase), then no warning will be thrown. Otherwise, a warning will be thrown. This rule only flags definitions and assignments but not function calls.
 
+### Options
+
+This rule accepts a single options argument with the following defaults:
+
+```js
+{
+    "rules": {
+        "camelcase": [2, {"properties": "always"}]
+    }
+}
+```
+
+`Properties` can have the following values:
+1. `always` is the default and checks all proprty names
+2. `never` does not check property names at all
+3. `nomethods` does not check property names if the object contains at least one member function
```

Comment:
We never came to a resolution on this third option. Might be better to just stick with always and never for now.